src/utils/common.py

```diff
@@ -111,6 +113,13 @@ def _set_seed(seed: int):
         torch.cuda.manual_seed_all(seed)
 
 
+def char2int(text: list, char_set: set):
+    char2int_dict = {char: i + 1 for i, char in enumerate(char_set)}
+
+    targets = torch.LongTensor([char2int_dict[c] for c in text])
```

Comment:
Blank label must be 0, so you add 1 to all char ids.
But if you have blank symbol in the char set, there'll be two labels for blank symbol.